src/screens/Ticketing/Ticket/ValidityLine.tsx

```diff
@@ -8,39 +8,40 @@ import colors from '../../../theme/colors';
 const SPACE_BETWEEN_VERTICAL_LINES = 72;
 const AnimatedLinearGradient = Animated.createAnimatedComponent(LinearGradient);
 
-const ValidityLine: React.FC<{
-  isValid: boolean;
-  nowSeconds: number;
-  validFrom: number;
-  validTo: number;
-}> = ({isValid, nowSeconds, validFrom, validTo}) => {
+type Props =
+  | {
+      status: 'valid';
+      nowSeconds: number;
+      validFrom: number;
+      validTo: number;
+    }
+  | {
+      status: 'reserving';
+      nowSeconds?: never;
```

Comment:
Why `?: never` here and not just 

```ts
type Props =
  | {
      status: 'valid';
      nowSeconds: number;
      validFrom: number;
      validTo: number;
    }
  | {
      status: 'reserving';
     }
  | {
      status: 'expired';
     };
```

Reply:
Yeah, I'll remove it. I initially had `{ status, nowSeconds, validFrom, validTo}: Props`, which was not okay if omitting 'never' as it said _"... does not exist on type 'Props'"_.